### DISCUSSION TITLE
Is it possible to reset the admin password via command line?

### DISCUSSION THREAD

**Discussion by NSFWHubSocial** [mastodon/mastodon#21912] [Q&A] [answered]
I did a fresh 4.0.2 install but I didn't see the auto generated admin password while the SSL certificate was installing and I'm not receiving the reset password email for some reason (security feature perhaps?) so my question is, can I reset the admin password via command line?

**Comment by NSFWHubSocial** [comment, marked as answer]
I got this fixed.

**Reply by naught101** [reply]
How?

**Reply by justdaj** [reply]
See here, for command line

https://docs.joinmastodon.org/admin/tootctl/#accounts-modify

**Reply by naught101** [reply]
I found this and posted he commands I used here: https://github.com/mastodon/mastodon/issues/22431
